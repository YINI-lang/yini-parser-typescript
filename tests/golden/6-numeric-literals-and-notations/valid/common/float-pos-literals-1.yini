^ PosFloat
float1 = 0.0                // smallest non-negative float
float2 = 1.0                // smallest positive whole number as float
float3 = 3.4028234663852886 // approx max 32-bit float (IEEE 754 single precision)
float4 = 340282346638528860
float5 = 9999999999999999.0 // large float near 53-bit integer precision limit

^ PosFloat2
float1 = +0.0                // smallest non-negative float
float2 = +1.0                // smallest positive whole number as float
float3 = +3.4028234663852886 // approx max 32-bit float (IEEE 754 single precision)
float4 = +340282346638528860
float5 = +9999999999999999.0 // large float near 53-bit integer precision limit
