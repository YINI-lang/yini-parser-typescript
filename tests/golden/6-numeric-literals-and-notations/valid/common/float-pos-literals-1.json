{
    "PosFloat": {
        "float1": 0.0,
        "float2": 1.0,
        "float3": 3.4028234663852886,
        "float4": 340282346638528860.5,
        "float5": 10000000000000000.0
    },
    "PosFloat2": {
        "float1": 0.0,
        "float2": 1.0,
        "float3": 3.4028234663852886,
        "float4": 340282346638528860.5,
        "float5": 10000000000000000.0
    }
}
